articles/governance/blueprints/overview.md

```diff
@@ -4,7 +4,7 @@ description: Understand how the Azure Blueprints service enables you to create,
 ms.date: 11/21/2019
 ms.topic: overview
 ---
-# What is Azure Blueprints?
+# What are Azure Blueprints?
```

Comment:
As Azure Blueprints is the name of a service, `is` is correct here.  Please revert.